src/content/learn/start-a-new-react-project.md

```diff
@@ -1,110 +1,112 @@
+**এই সমস্যাগুলি React এর নয়। এই কারণেই Svelte এর SvelteKit আছে, Vue এর Nuxt আছে ইত্যাদি।** এই সমস্যাগুলি নিজে থেকে সমাধান করতে, আপনাকে রাউটার এবং ডেটা আনার লাইব্রেরির সাথে আপনার বান্ডলারকে সংযোগ করতে হবে। একটি প্রাথমিক কার্যকারী সেটআপ করা কঠিন নয়, তবে একটি অ্যাপ তৈরির সাথে অনেক সূক্ষ্মতা জড়িত থাকে যা সময়ের সাথে সাথে অ্যাপের সাইজ বাড়লেও দ্রুত লোড করতে পারেতে হয়। আপনি একটি পৃষ্ঠা লোড করার জন্যে নুন্নতম কোড পাঠাতে চাইবেন কিন্তু পৃষ্ঠার ডাটা প্যারালাল ভাবে লোড একটি একক ক্লায়েন্ট-সার্ভার রাউন্ডট্রিপে করতে চান। ওয়েবসাইটের Progressive enhancement এর জন্যে আপনি সম্ভবত আপনার জাভাস্ক্রিপ্ট কোডটি চালানোর আগে পৃষ্ঠাটি ইন্টারেক্টিভ করতে চাইবেন। আপনি আপনার মার্কেটিং পেইজ গুলির জন্য সম্পূর্ণরূপে স্ট্যাটিক HTML ফাইলগুলির একটি ফোল্ডার তৈরি করতে চাইতে পারেন যা যে কোনও জায়গায় হোস্ট করা যেতে পারে এবং জাভাস্ক্রিপ্ট বন্ধ থাকেলও কাজ করতে পারে। এই ক্ষমতাগুলি অ্যাপের মধ্যে তৈরি করতে আসল কাজ লাগে। 
 
-**React frameworks on this page solve problems like these by default, with no extra work from your side.** They let you start very lean and then scale your app with your needs. Each React framework has a community, so finding answers to questions and upgrading tooling is easier. Frameworks also give structure to your code, helping you and others retain context and skills between different projects. Conversely, with a custom setup it's easier to get stuck on unsupported dependency versions, and you'll essentially end up creating your own framework—albeit one with no community or upgrade path (and if it's anything like the ones we've made in the past, more haphazardly designed).
+**কোন অতিরিক্ত কাজ ছাড়াই এই পৃষ্ঠার React ফ্রেমওয়ার্ক গুলো ডিফল্টভাবে এই ধরনের সমস্যাগুলি সমাধান করে।** তারা আপনাকে খুব সহজে শুরু করতে সাহায্য করে এবং তারপরে আপনার প্রয়োজনের সাথে আপনার অ্যাপটি স্কেল করতে সাহায্য করে। প্রতিটি React ফ্রেমওয়ার্কের একটি কমিউনিটি রয়েছে, তাই প্রশ্নের উত্তর খুঁজে পাওয়া এবং টুলিং আপগ্রেড করা সহজ। ফ্রেমওয়ার্কগুলি আপনার কোডের কাঠামোও দেয়, আপনাকে এবং অন্যদেরকে বিভিন্ন প্রকল্পের মধ্যে প্রসঙ্গ এবং দক্ষতা বজায় রাখতে সাহায্য করে। বিপরীতভাবে, একটি কাস্টম সেটআপের মধ্যে খুব সহজেই unsupported dependency version গুলিতে আটকে যাওয়া সহজ, এবং আপনি মূলত আপনার নিজস্ব কাঠামো তৈরি করতে পারবেন—যদিও কোনো framework আপগ্রেড পথ নেই (এবং যদি এটি আরও এলোমেলোভাবে ডিজাইন করা আমাদের অতীতে তৈরি করা framework গুলির মতো কিছু হয়)।
```

Comment:
ভাই, এই ট্রান্সলেশনটা পারলে আবার একটু রিভিউ করবেন,
and you'll essentially end up creating your own framework—albeit one with no community or upgrade path

কিছু যেন মিসিং মনে হচ্ছে বুজতে পারছি না, আমার ভুল ও হতে পারে।